### PR TITLE
docs(gcp): ai platform pipelines use emissary tooltip update

### PR DESCRIPTION
**Pull request by Bobgy**
**Description of your changes:**
Part of #6294 

before the PR, it looked like the following on the UI, basically the format doesn't support markdown, so I have to reduce the number of links and what content to include here:
image

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: manifests/gcp_marketplace/schema.yaml [hunk 1]
```diff
@@ -175,24 +175,20 @@ properties:
     default: false
     description: |-
       Kubeflow Pipelines default to use docker executor. Select this option to
-      use [the emissary executor (Alpha)](https://github.com/kubeflow/pipelines/issues/6249).
+      use the emissary executor.
 
-      Refer to migrating to emissary executor documentation for migration instructions.
+      Warning, docker executor does not work on Google Kubernetes Engine (GKE) 1.19+
```
Comment (shuesc1) [inline manifests/gcp_marketplace/schema.yaml:180]: "Docker executor does not work on Google Kubernetes Engine (GKE) 1.19+ by default, as docker container runtime is no longer the default container runtime; docker container runtime will be deprecated on GKE 1.20+."

File: manifests/gcp_marketplace/schema.yaml [hunk 2]
```diff
@@ -175,24 +175,20 @@ properties:
...
+      by default, because docker container runtime is no longer the default. Furthermore,
+      docker container runtime will be deprecated on GKE 1.20+.
 
-      Emissary executor is more portable. On Google Kubernetes Engine (GKE)
-      clusters 1.19+, containerd is the default container runtime instead of
-      docker, so docker executor does not work properly. Emissary executor is not
-      coupled to a specific container runtime, so it works properly on any
-      Kubernetes clusters including GKE 1.19+ with containerd as runtime.
+      To prepare for the migration to emissary executor, refer to [1] for more details.
```
Comment (shuesc1) [inline manifests/gcp_marketplace/schema.yaml:184]: ".....refer to [1] for more details." -> "refer to [Argo Workflow Executors][1] for more details."

File: manifests/gcp_marketplace/schema.yaml [hunk 3]
```diff
@@ -175,24 +175,20 @@ properties:
...
-      (deprecated) as the node image. For more information, refer to
-      [GKE Node Images](https://cloud.google.com/kubernetes-engine/docs/concepts/node-images).
+      To keep using docker executor for GKE 1.19+, configure your node pools to
+      use docker as container runtime by selecting "Container-Optimized OS with Docker (cos)" (deprecated)
+      as the node image [2].
 
-      Although emissary is in Alpha, Kubeflow Pipelines team is actively helping
-      to make it mature. Welcome any feedback if you have already tried emissary
-      executor: https://github.com/kubeflow/pipelines/issues/6249.
+      [1] https://www.kubeflow.org/docs/components/pipelines/installation/choose-executor/
```
Comment (shuesc1) [inline manifests/gcp_marketplace/schema.yaml:190]: [1]:
[2]:

File: manifests/gcp_marketplace/schema.yaml [hunk 4]
```diff
@@ -175,24 +175,20 @@ properties:
...
-      coupled to a specific container runtime, so it works properly on any
-      Kubernetes clusters including GKE 1.19+ with containerd as runtime.
+      To prepare for the migration to emissary executor, refer to [1] for more details.
 
-      If you want to keep using docker executor for GKE 1.19+, you need to configure
-      your node pools to use docker as container runtime by selecting "Container-Optimized OS with Docker (cos)"
-      (deprecated) as the node image. For more information, refer to
-      [GKE Node Images](https://cloud.google.com/kubernetes-engine/docs/concepts/node-images).
+      To keep using docker executor for GKE 1.19+, configure your node pools to
+      use docker as container runtime by selecting "Container-Optimized OS with Docker (cos)" (deprecated)
```
Comment (shuesc1) [inline manifests/gcp_marketplace/schema.yaml:187]: "...use docker as a container runtime by selecting "Container-Optimized OS with Docker (cos)" (deprecated) as the [node image][2]."

**Comment by kramachandran** [comment]
This feels way to long to be a tool tip -- can we shorten it?  Just focus on two facts - you need Emissary if you want to GKE 1.19+; and that it causes a breaking change?

**Comment by Bobgy** [comment]
Thank you for the review!
Let me try to resolve Karthik's comment first.

> This feels way too long to be a tool tip -- can we shorten it? Just focus on two facts - you need Emissary if you want to GKE 1.19+; and that it causes a breaking change?

I think this is a great advice. For clarification, on GKE 1.19+, people can configure node image to use docker container runtime + docker executor too.

What do you think about rough ideas of the following?
```
Warning, Kubeflow Pipelines default to use docker executor, but docker executor does not work on
Google Kubernetes Engine (GKE) 1.19+ by default. You cluster nodes must use docker as container runtime.

To use emissary executor, some of your pipelines may require manual migrations.

Refer to Argo Workflow Executors documentation for detailed instructions:
https://www.kubeflow.org/docs/components/pipelines/installation/choose-executor/
```

**Comment by karkumar** [comment]
Thanks the updated text seems much clearer.

**Pull request merged**